indico/modules/events/abstracts/templates/management/overview.html

```diff
@@ -14,33 +14,53 @@
             <div class="icon icon-upload"></div>
             <div class="text">
                 <div class="label">
-                    {% trans %}Submission{% endtrans %}
+                    {%- trans %}Submission{% endtrans -%}
                 </div>
-                {% trans %}Configure the submission dates, the abstract fields, etc.{% endtrans %}
+                {%- trans %}Configure general submission settings and email notification templates.{% endtrans -%}
             </div>
             <div class="toolbar">
                 <div class="group">
                     <a href="#" class="i-button icon-mail"
                        data-href="{{ url_for('.email_tpl_list', event) }}"
                        data-title="{% trans %}Configure e-mail notifications{% endtrans %}"
                        data-ajax-dialog>
-                        {% trans %}Notifications{% endtrans %}
+                        {%- trans %}Notifications{% endtrans -%}
                     </a>
                 </div>
                 <div class="group">
-                    <a href="#" class="i-button icon-list js-dialog-action"
-                       data-title="{% trans %}Manage fields{% endtrans %}"
-                       data-href="{{ url_for('contributions.manage_fields', event) }}"
+                    <a href="#" class="i-button icon-settings"
+                       data-href="{{ url_for('.manage_submission_settings', event) }}"
+                       data-title="{% trans %}Configure abstract submission{% endtrans %}"
                        data-ajax-dialog>
-                        {% trans %}Abstract fields{% endtrans %}
+                        {%- trans %}Settings{% endtrans -%}
                     </a>
                 </div>
+            </div>
+        </div>
+
+        <div class="section">
+            <div class="icon icon-wrench"></div>
+            <div class="text">
+                <div class="label">
+                    {%- trans %}Fields and types{% endtrans -%}
+                </div>
+                {%- trans %}Configure the contribution types and abstract fields.{% endtrans -%}
+            </div>
+            <div class="toolbar">
                 <div class="group">
-                    <a href="#" class="i-button icon-settings"
-                       data-href="{{ url_for('.manage_submission_settings', event) }}"
-                       data-title="{% trans %}Configure abstract submission{% endtrans %}"
+                    <a href="#" class="i-button"
+                       data-href="{{ url_for('contributions.manage_types', event) }}"
+                       data-title="{% trans %}Manage contribution types{% endtrans %}"
+                       data-ajax-dialog>
+                        {%- trans %}Contribution types{% endtrans -%}
```

Comment:
Not the only ones ("Close" call for abstracts also doesn't), but I believe we're overusing icons... Any better idea?